test/addons-napi/test_general/test.js

```diff
@@ -34,3 +34,12 @@ assert.ok(test_general.testGetPrototype(baseObject) !==
 // test version management functions
 // expected version is currently 1
 assert.strictEqual(test_general.testGetVersion(), 1);
+
+[123, 'test string', function() {}, new Object(), true,
+ undefined, Symbol()].forEach((val) => {
+   assert.strictEqual(typeof val, typeof (test_general.testNapiTypeof(val)));
```

Comment:
That would probably work as well.  Just not what I can up with.  I'll take a look as its probably an easy change.